### DISCUSSION TITLE
Save .aab format

### DISCUSSION THREAD

**Discussion by albeci** [kefir500/apk-editor-studio#42] [Ideas]
Open the .apk file and save it in .aab format
Would this possibility exist?

**Comment by kefir500** [comment]
The related question has already been answered [here](https://github.com/kefir500/apk-editor-studio/issues/3#issuecomment-487349542). The main point regarding the toolchain is still the same. However, it's pretty obvious that the `.aab` format *is* the future, at least when it comes to Google Play.

Currently, I'm working on the following tasks:
- Moving from `qmake` to `CMake`
- Moving from `QSyntaxHighlighter` to `KSyntaxHighlighting`

When I finish these tasks, I'll try to dig more information on this topic and see if there are any ways to implement the `.aab` support without bloating the toolchain and third-party dependencies (such as duplicated `AAPT2`, for example). Until then, I'm tracking [this discussion](https://github.com/iBotPeaches/Apktool/discussions/2572) in case the Apktool (which is used under the hood) team will implement the `.aab` support.

**Comment by henriquelucas** [comment]
Hello everybody.
Good morning, good afternoon or good evening depending on the time you are reading this comment.

Is there any more news on this topic? Is there any way to convert from .apk to .aab

**Reply by kefir500** [reply]
No news, unfortunately.
I don't have much spare time to explore and implement this functionality for APK Editor Studio.
However, you can always perform the AAB build manually as described [here](https://developer.android.com/studio/build/building-cmdline#bundletool-build).
